app/controllers/shoutouts_controller.rb

```diff
@@ -55,16 +55,20 @@ def create
     if shoutout.tag_list.include?("london")
       slack.post_message(Settings.slack.london, cheered_notification_message_for_channel(message))
     end
-
     if shoutout.tag_list.include?("capetown")
       slack.post_message(Settings.slack.cape_town, cheered_notification_message_for_channel(message))
     end
 
     render text: "You cheered for #{shoutout.recipients.join(' & ')}! Visit #{root_url} to see your cheer."
+    slack.post_direct_message(Settings.slack.token, shoutout.recipients, cheered_notification_message_for_user)
   end
 
   private
 
+  def cheered_notification_message_for_user
+    "Lucky you, you've been cheered. Would you like to cheer in return?"
```

Comment:
Did you folks agree on a different wording? This is (slightly) different to the Trello card..